templates/taxi/driver_create_or_update.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Create driver:</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="Save" class="btn btn-primary">
+    <a href="{% url 'taxi:driver-list' %}" class="btn btn-secondary ml-3">
```

Comment:
Ensure that the URL name 'taxi:driver-list' used in the cancel button matches the URL configuration in your Django project. If the URL pattern is named differently, update this to prevent navigation errors.